src/sentry/models/dashboard.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+from django.utils import timezone
+from jsonfield import JSONField
+
+from sentry.constants import ObjectStatus
+from sentry.db.models import BoundedPositiveIntegerField, FlexibleForeignKey, Model, sane_repr
+
+
+class Dashboard(Model):
+    """
+    A dashboard.
+    """
+    __core__ = True
+
+    title = models.CharField(max_length=128)
```

Comment:
This might be too short for some users. Going to 255 would give more space with no additional storage cost.